projects/observability/src/shared/dashboard/widgets/donut/donut-widget.model.ts

```diff
@@ -65,13 +69,25 @@ export class DonutWidgetModel {
   })
   public header?: WidgetHeaderModel;
 
+  @ModelProperty({
+    key: 'color-palette',
+    type: ARRAY_PROPERTY.type,
+    required: false
+  })
+  public colorPalette: string[] = [];
```

Comment:
A color palette should also accept a string as key - the color service reflects this scenario. Since we don't have unions in our crude type system, can just leave that as `UNKNOWN_PROPERTY.type` - at some point, we'll create a color property.